sdk/identity/azure-identity/README.md

```diff
@@ -45,7 +45,7 @@ Maven dependency for Azure Secret Client library. Add it to your project's pom f
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-identity</artifactId>
-    <version>1.0.4</version>
+    <version>1.1.0-beta.2</version>
```

Comment:
This is more a conceptual question, do we want the READMEs to use the latest released version even if it is a beta or the latest GA'd version?

@JonathanGiles @joshfree

Reply:
Readme should always point to the GA version.

Reply:
updated